db/schema_tables.cc

```diff
@@ -2353,6 +2353,7 @@ schema_ptr create_table_from_mutations(const schema_ctxt& ctxt, schema_mutations
 
     if (auto partitioner = sm.partitioner()) {
         builder.with_partitioner(*partitioner, smp::count, ctxt.murmur3_partitioner_ignore_msb_bits());
+        builder.with_sharding_info(smp::count, ctxt.murmur3_partitioner_ignore_msb_bits());
```

Comment:
But doesn't the partitioner already contain a sharding_info? Either we should drop the sharding_info from the partitioner, or use the sharding_info from the partitioner. It doesn't make sense for one table to have two sharding_infos.

Reply:
Ok, I see this later pach removed sharding_info from the partitioner. So it's good.

Reply:
And the changelog explains the intent, so I can't complain.